packages/@aws-cdk/aws-lambda-event-sources/lib/sqs.ts

```diff
@@ -14,6 +14,16 @@ export interface SqsEventSourceProps {
    */
   readonly batchSize?: number;
 
+  /**
+   * The maximum amount of time to gather records before invoking the function.
+   * Maximum of Duration.minutes(5)
+   *
+   * Valid Range: Minimum value of 0 minutes. Maximum value of 5 minutes.
+   *
+   * @default - no batching window. The lambda function will be invoked immediately with the records that are available.
+   */
+  readonly maxBatchingWindow?: Duration;
```

Comment:
Just `batchingWindow` is sufficient. Analogous to `batchSize`.

Reply:
Looks like we're calling it `maxBatchingWindow` elsewhere already - https://github.com/aws/aws-cdk/blob/b9407102304f043adcd9a4fc1cde4d23d3da9004/packages/%40aws-cdk/aws-lambda-event-sources/lib/stream.ts#L73-L79.

Ignore this comment. It's fine to leave it as is.